website/landing/components/Api/Api.tsx

```diff
@@ -0,0 +1,231 @@
+import { content } from "../../content/api-section";
+import { Box, List, SandpackLogo, Text, TextHighlight } from "../common";
+
+const ContentLandmark = () => {
+  return (
+    <Box
+      css={{
+        alignItems: "center",
+        background: "$surface",
+        color: "white",
+        display: "flex",
+        height: "595px",
+        justifyContent: "center",
+        margin: "0 auto",
+        width: "343px",
+
+        "@bp1": {
+          width: "376px",
+        },
+
+        "@bp2": {
+          height: "540px",
+          width: "360px",
+        },
+
+        "@bp3": {
+          height: "720px",
+          width: "480px",
+        },
+      }}
+    >
+      <Text>content landmark.</Text>
+    </Box>
+  );
+};
+
+export const Api: React.FC = () => {
+  return (
+    <Box
+      as="section"
+      css={{
+        alignItems: "center",
+        background: "$lightBackground",
+        color: "$lightTextPrimary",
+        display: "flex",
+        flexDirection: "column",
+        gap: "100px",
+        padding: "200px 16px 100px",
+      }}
+    >
+      <Box
+        css={{
+          alignItems: "center",
+          display: "flex",
+          flexDirection: "column",
+          gap: "40px",
+        }}
+      >
+        <Box
+          css={{
+            width: "60px",
+            "@bp1": {
+              width: "100px",
+            },
+            "@bp2": {
+              width: "120px",
+            },
+            "@bp3": {
+              width: "150px",
+            },
+          }}
+        >
+          <SandpackLogo />
+        </Box>
```

Comment:
The `SandpackLogo` keeps the same width per breakpoint, maybe the space definitions should live with the logo itself rather than with individual wrappers.